Yoast/wpseo-woocommerce
# Change javascript linters to eslint

## Summary

This PR can be summarized in the following changelog entry:

* Removes the jshint and jsvalidate JavaScript checkers in favor of eslint.

## Relevant technical choices:

* Removing the jshint and jsvalidate JavaScript checkers in favor of eslint.

## Test instructions

This PR can be tested by following these steps:

1. Ensure you have all development dependencies for the project by running `yarn install`.
2. Execute the command `grunt check`, this is an alias that contains the following grunt commands: 
  - 'eslint'
  - 'jsonlint'
  - 'jscs'
  - 'phplint'
  - 'phpcs'
  - 'checktextdomain'
3. Verify that the new JavaScript checkers do their job.

Fixes #228
